### PR TITLE
Add missing Abelian groups docs

### PR DESCRIPTION
**Pull request by lgoettgens**
Similarly to the update in Hecke:
https://github.com/thofma/Hecke.jl/pull/1631

(this is a copy of https://github.com/oscar-system/Oscar.jl/pull/4184, but this time not from a fork to get a preview)

preview: https://docs.oscar-system.org/previews/PR4186

### CODE REVIEW

**Comment by lgoettgens** [comment]
This is still broken, since I just cherry-picked the branch:
image

File: docs/doc.main [hunk 1]
```diff
@@ -24,6 +24,9 @@
      "Groups/autgroup.md",
      "Groups/grouplib.md",
      "Hecke/manual/abelian/introduction.md",
+     "Hecke/manual/abelian/elements.md",
+     "Hecke/manual/abelian/maps.md",
+     "Hecke/manual/abelian/structural.md",
```
Comment (lgoettgens) [inline docs/doc.main]:
```suggestion
     "Abelian Groups" => [
        "Introduction" => "Hecke/manual/abelian/introduction.md",
        "Elements" => "Hecke/manual/abelian/elements.md",
        "Morphisms" => "Hecke/manual/abelian/maps.md",
        "Structural Computations" => "Hecke/manual/abelian/structural.md",
     ],
```

**Comment by lgoettgens** [comment]
I think the change in [`4e95637` (#4186)](https://github.com/oscar-system/Oscar.jl/pull/4186/commits/4e95637f086aa174826c29f1d4900ff47ad33a8f) makes it behave like in the Hecke navigation bar.

cc @paemurru

**Comment by lgoettgens** [comment]
A local build now looks like this:
image
The preview should look the same (once the CI job finishes).

Is this what you imagined @paemurru ?

**Review by thofma — APPROVED** [review]

**Pull request merged**